bee-node/bee-node/Cargo.toml

```diff
@@ -38,6 +38,7 @@ futures = { version = "0.3.17", default-features = false }
 fxhash = { version = "0.2.1", default-features = false }
 hex = { version = "0.4.3", default-features = false }
 iota-crypto = { version = "0.9.1", default-features = false, features = [ "ed25519", "random", "blake2b" ] }
+inx = { version = "0.1.0", git = "https://github.com/pvdrz/inx", branch = "reexport-server", default-features = false }
```

Comment:
I guess this dep should be against `iotaledger/inx`, right? Maybe open a PR also to merge your branch?

Reply:
yeah I'm waiting for any further changes I need to do to the `inx` repo first.